ui/src/logs/components/LogsTable.tsx

```diff
@@ -270,7 +315,7 @@ class LogsTable extends Component<Props, State> {
     if (this.isClickable(column)) {
       return (
         <div
-          className={classnames('logs-viewer--cell', {
+          className={classnames(`logs-viewer--cell`, {
```

Comment:
this could be single quotes

Reply:
good catch, was interpolating a variable at one point and it didn't get switched back.